### PR TITLE
cameras: Convert to es6.

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

File: src/core/EventDispatcher.js [hunk 1]
```diff
@@ -2,11 +2,9 @@
  * https://github.com/mrdoob/eventdispatcher.js/
  */
 
-function EventDispatcher() {}
+class EventDispatcher {
```
Comment (Mugen87) [inline src/core/EventDispatcher.js:5]: `EventDispatcher` should not be part of this PR.

Comment (linbingquan) [reply src/core/EventDispatcher.js]: > `EventDispatcher` should not be part of this PR. Better to put this change in your `core` PR.

Thank you review, Let me try.

File: src/cameras/OrthographicCamera.js [hunk 2]
```diff
@@ -134,7 +130,8 @@ OrthographicCamera.prototype = Object.assign( Object.create( Camera.prototype ),
 
 	}
 
-} );
+}
 
+OrthographicCamera.prototype.isOrthographicCamera = true;
```
Comment (marcofugaro) [inline src/cameras/OrthographicCamera.js:135]: could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

Comment (linbingquan) [reply src/cameras/OrthographicCamera.js]: > could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

OK.

**Comment by DefinitelyMaybe** [comment]
@linbingquan could you please sort out the conflicts with this PR?

**Comment by linbingquan** [comment]
> @linbingquan could you please sort out the conflicts with this PR?

Thank you  for your tip.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!
